src/components/sbb-link/sbb-link.tsx

```diff
@@ -160,21 +176,22 @@ export class SbbLink implements LinkButtonProperties, ComponentInterface {
 
   public render(): JSX.Element {
     let TAG_NAME: string;
-    let attributeList: object = getLinkButtonBaseAttributeList(
-      this.idValue,
-      this._getClassString(),
-      this
-    );
+    let attributeList: object = getLinkButtonBaseAttributeList(this);
```

Comment:
Is it not possible to use `Record<string, string>` here?
```suggestion
    let attributeList: Record<string, string> = getLinkButtonBaseAttributeList(this);
```